src/dataflow-types/src/types.rs

```diff
@@ -454,6 +454,7 @@ pub enum Envelope {
     None,
     Debezium(DebeziumDeduplicationStrategy),
     Upsert(DataEncoding),
+    CdcV2,
```

Comment:
if its going to be called `envelope materialize` externally should it be called that within the codebase as well? i agree with nikhil that cdcv2 was / is forever confusing to me because cdcv1 was never a thing